libs/components/angular-tree-component/src/lib/modules/angular-tree/angular-tree-wrapper.component.html

```diff
@@ -1,7 +1,5 @@
 <div
   class="sky-angular-tree-wrapper"
-  role="tree"
```

Comment:
We were experiencing another accessibility violation because we're including a toolbar under the `tree` role. We should move the toolbar outside the tree, instead.